tooling/nargo_cli/src/cli/mod.rs

```diff
@@ -52,6 +53,7 @@ pub(crate) struct NargoConfig {
 enum NargoCommand {
     Backend(backend_cmd::BackendCommand),
     Check(check_cmd::CheckCommand),
+    Fmt(fmt_cmd::FormatCommand),
```

Comment:
```suggestion
    #[command(hide = true)] // Hidden while the feature has not been extensively tested
    Fmt(fmt_cmd::FormatCommand),
```

Reply:
Lets make the feature there but keep it hidden while we test it out a bit and make modifications